src/events.h

```diff
@@ -23,135 +23,75 @@ enum class EventInfoId
-	bool eventPlayerOnLookInShop(Player* player, const ItemType* itemType, uint8_t count);
-	bool eventPlayerOnLookInMarket(Player* player, const ItemType* itemType);
-	ReturnValue eventPlayerOnMoveItem(Player* player, Item* item, uint16_t count, const Position& fromPosition,
-	                                  const Position& toPosition, Cylinder* fromCylinder, Cylinder* toCylinder);
-	void eventPlayerOnItemMoved(Player* player, Item* item, uint16_t count, const Position& fromPosition,
-	                            const Position& toPosition, Cylinder* fromCylinder, Cylinder* toCylinder);
-	bool eventPlayerOnMoveCreature(Player* player, Creature* creature, const Position& fromPosition,
-	                               const Position& toPosition);
-	void eventPlayerOnReportRuleViolation(Player* player, const std::string& targetName, uint8_t reportType,
-	                                      uint8_t reportReason, const std::string& comment,
-	                                      const std::string& translation);
-	bool eventPlayerOnReportBug(Player* player, const std::string& message, const Position& position, uint8_t category);
-	void eventPlayerOnRotateItem(Player* player, Item* item);
-	bool eventPlayerOnTurn(Player* player, Direction direction);
-	bool eventPlayerOnTradeRequest(Player* player, Player* target, Item* item);
-	bool eventPlayerOnTradeAccept(Player* player, Player* target, Item* item, Item* targetItem);
-	void eventPlayerOnTradeCompleted(Player* player, Player* target, Item* item, Item* targetItem, bool isSuccess);
-	void eventPlayerOnPodiumRequest(Player* player, Item* item);
-	void eventPlayerOnPodiumEdit(Player* player, Item* item, const Outfit_t& outfit, bool podiumVisible,
-	                             Direction direction);
-	void eventPlayerOnGainExperience(Player* player, Creature* source, uint64_t& exp, uint64_t rawExp, bool sendText);
-	void eventPlayerOnLoseExperience(Player* player, uint64_t& exp);
-	void eventPlayerOnGainSkillTries(Player* player, skills_t skill, uint64_t& tries);
-	void eventPlayerOnWrapItem(Player* player, Item* item);
-	void eventPlayerOnInventoryUpdate(Player* player, Item* item, slots_t slot, bool equip);
-	void eventPlayerOnNetworkMessage(Player* player, uint8_t recvByte, NetworkMessage* msg);
-	bool eventPlayerOnSpellCheck(Player* player, const Spell* spell);
-
-	// Monster
-	void eventMonsterOnDropLoot(Monster* monster, Container* corpse);
-	bool eventMonsterOnSpawn(Monster* monster, const Position& position, bool startup, bool artificial);
-
-	int32_t getScriptId(EventInfoId eventInfoId)
-	{
-		switch (eventInfoId) {
-			case EventInfoId::CREATURE_ONHEAR:
-				return info.creatureOnHear;
-			case EventInfoId::MONSTER_ONSPAWN:
-				return info.monsterOnSpawn;
-			default:
-				return -1;
-		}
-	};
-
-private:
-	LuaScriptInterface scriptInterface;
-	EventsInfo info;
-};
+namespace tfs::events {
+
+bool load();
```

Comment:
I prefer to change this for each events::x, as it gives us better control over each type. However, this approach will result in repeated code and reading the same file multiple times. I suggest this because I plan to add `global events` and `move events` to the namespace.